doc/releases/changelog-0.42.0.md

```diff
@@ -135,20 +134,71 @@ TODO
 
 <h4>Qualtran integration 🔗</h4>
```

Comment:
I'm a bit confused why there's two separate sections that demonstrate how to use `to_bloq` 🤔. I really like the first three bullet points (wrap PL circuits..., use smart default mapping..., custom user-defined mapping...), but we go back to explaining the function arguments and then there are more examples. I think all of the examples shown are nice, but maybe the ordering is a bit jumbled?

Reply:
Ah, forgot to delete one :facepalm: